ast35/Parser/Python.asdl

```diff
@@ -70,7 +70,7 @@ module Python
          -- x < 4 < 3 and (x < 4) < 3
          | Compare(expr left, cmpop* ops, expr* comparators)
          | Call(expr func, expr* args, keyword* keywords)
-         | Num(object n) -- a number as a PyObject.
+         | Num(object n, int? underscores) -- a number as a PyObject.
```

Comment:
I think it'd be a bit clearer if this was named `contains_underscores` instead.  Also, could you add a comment noting that this field is not part of standard Python and its purpose is to signal that a newer Python version feature was used?